src/App/KickOff.php

```diff
@@ -48,6 +48,20 @@ public function index($configFile) {
         return $this->errorCount;
     }
 
+    public function seocheck($wepbage)
```

Comment:
Since this method is nearly a duplicate of the index method I'd prefer to keep one entry point for this class. This would conform best to the SOLID principle.
From the SeoCheckCommand the index-method could be called with the configFile beeing injected. The webpage parameter could be added as optional parameter to the index method.